spring-cloud-task-samples/multiple-datasources/README.adoc

```diff
@@ -21,9 +23,25 @@ which one to be used for the Spring Cloud Task repository.
 $ mvn clean package
 ----
 
-== Run:
+== Execute sample using 2 embedded databases (default):
 
 [source,shell,indent=2]
 ----
 $ java -jar target/multiple-datasources-2.2.2.RELEASE.jar
```

Comment:
This is not included in the PR, but the version of the jar in the command should be the same whether we use the `embedded` or `external` profile.

Reply:
Updated.   Thanks for catching